src/main/java/eu/eutampieri/catacombs/ui/utils/FontUtils.java

```diff
@@ -6,8 +6,7 @@
 public class FontUtils {
 	
 	public static int getTextWidth(Font font, String text) {
-		return (int) font.getStringBounds(text, new FontRenderContext(font.getTransform(),
-				true, true)).getWidth();
+		return (int) font.getStringBounds(text, new FontRenderContext(font.getTransform(), true, true)).getWidth();
```

Comment:
Maybe add a few variables for clarity to break this one-liner

Reply:
fixed but in issue_6_utils_loaders

Reply:
Well, we’ll have to wait for the pr to be merged